src/modules/markets/components/markets-view.jsx

```diff
@@ -66,6 +68,14 @@ export default class MarketsView extends Component {
   }
 
   componentWillUpdate(nextProps, nextState) {
+    const newTags = decodeURIComponent(parseQuery(nextProps.location.search)[TAGS_PARAM_NAME] || '')
```

Comment:
No need for the `decodeURIComponent` here.

Also, I’d do a conditional around just `location.search` as parsing that is a performance hit on each update.

Reply:
i was finding there was a need as tags such as `US Politics` where coming in URI encoded like so `US%20Politics`